nanofi/include/sitetosite/CRawSocketProtocol.h

```diff
@@ -76,23 +76,15 @@ int16_t sendPacket(struct CRawSiteToSiteClient * client, const char * transactio
 
 CTransaction* createTransaction(struct CRawSiteToSiteClient * client, TransferDirection direction);
 
-static const char * getResourceName() {
+static inline const char * getResourceName() {
```

Comment:
Why all the explicit inlining?

Reply:
According to @arpadboda the static function definitions in the headers were introduced to bypass link issues, not an actual solution to the problem. They cause `-Wunused-function` as they should be referenced in every single translation unit where they are included. The original link issues should be resolved in the future, but as this codebase is abandoned at the moment, the easiest solution was to mark the static functions inline, which bypasses the warning.